partner_multi_image/hooks.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+# © 2009 Sharoon Thomas Open Labs Business Solutions
+# © 2014 Serv. Tecnol. Avanzados Pedro M. Baeza
+# © 2015 Antiun Ingeniería S.L. - Jairo Llopis
+# © 2016 Sodexis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp.addons.base_multi_image.hooks import pre_init_hook_for_submodules
```

Comment:
You are missing [the uninstall hook](https://github.com/OCA/server-tools/blob/9.0/base_multi_image/hooks.py#L66).